codecov.yaml

```diff
@@ -0,0 +1,26 @@
+codecov:
+  require_ci_to_pass: true
```

Comment:
Is this the same meaning as `fail_ci_if_error`? :thinking:

Reply:
The meaning is the same as `fail_ci_if_error`, but this option must be used in this action.
This is because the return code of this action changes depending on this flag.